integration-junit4/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+dockerunit.greeting=
```

Comment:
missing newline (also why are we declaring this empty?)

Reply:
that's on purpose as I want the logic to pick the default value and then add a test that mounts a different `application.properties`.